src/inmanta/data/__init__.py

```diff
@@ -6624,11 +6624,13 @@ async def start_engine(
     pool_size: int = 10,
     max_overflow: int = 0,
     pool_timeout: float = 60.0,
-    echo: bool = False,
 ) -> None:
     """
     Start the SQL Alchemy engine for this process
     """
+    # Configure logging to emit sqlalchemy logs without relying on the echo flag.
+    # The echo flag also writes to stdout which results in duplicate logs
+    logging.getLogger("sqlalchemy.engine").setLevel(logging.INFO)
```

Comment:
Is this the correct place to have this?

Reply:
We have configurable logging on iso9. We should not longer hard code logging config. I would add this to the relevant logging config files in `inmanta-core/misc/`